MMM-WienerLinien.js

```diff
@@ -201,7 +297,7 @@ Module.register('MMM-WienerLinien', {
     },
 
     appendIncidentData(appendTo, type) {
-        for (let i = 0; i < this[type].length; i += 1) {
+        for (var i = 0; i < this[type].length; i += 1) {
```

Comment:
Why do we move to `var` from `let`?

Reply:
I am using an old ipad as magic mirror device. And the old safari version does not work with `let`, this is why I use `var`s. (on newer browsers it also works with `let`)